### PR TITLE
cmd, common, eth, server: show price per pixel in CLI

### PR DESCRIPTION
**Pull request by kyriediculous**
**What does this pull request do? Explain your changes. (required)**
Show price per pixel in CLI when picking `3. Show registered orchestrators`.

The price per pixel is queried from the database so it actually includes the transaction cost overhead of 1%. 

**How did you test each of these updates (required)**
Checked that the pricing of my mainnet orch is shown correctly (45+45/100)

**Checklist:**
- [ ] README and other documentation updated
- [x] Node runs in OSX and devenv
- [x] All tests in `./test.sh` pass

### CODE REVIEW

File: cmd/livepeer_cli/wizard_bond.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func (w *wizard) registeredOrchestratorStats() map[int]common.Address {
 	fmt.Println("+------------------------+")
 
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"ID", "Address", "Active", "Delegated Stake", "Reward Cut (%)", "Fee Share (%)", "Service URI"})
+	table.SetHeader([]string{"ID", "Address", "Active", "Delegated Stake", "Reward Cut (%)", "Fee Share (%)", "Service URI", "Price per pixel"})
```
Comment (yondonfu) [inline cmd/livepeer_cli/wizard_bond.go]: "Price Per Pixel" to match the capitalization convention in the other column names?

File: common/util.go [hunk 2]
```diff
@@ -226,6 +226,10 @@ func BaseTokenAmountToFixed(baseAmount *big.Int) (int64, error) {
 	return ratToFixed(rat, 100000)
 }
 
+func FixedToPrice(price, scalingFactor int64) *big.Rat {
```
Comment (yondonfu) [inline common/util.go]: Since this function is used specifically for converting fixed point numbers to prices (which have a hardcoded scaling factor of 1000 right now) can we omit the `scalingFactor` param and hardcode the scaling factor value in the function body similar to what we do in `PriceToFixed`?

IMO it would also be nice to place this function next to `PriceToFixed()` since the two functions are the inverse of each other.

File: server/webserver.go [hunk 3]
```diff
@@ -912,6 +912,16 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 				return
 			}
 
+			for _, o := range orchestrators {
+				dbO, err := s.LivepeerNode.Database.SelectOrchs(&lpcommon.DBOrchFilter{
+					Addresses: []common.Address{o.Address},
+				})
+				if err != nil {
+					continue
```
Comment (yondonfu) [inline server/webserver.go:951]: Let's log the error.

File: server/webserver.go [hunk 4]
```diff
@@ -912,6 +912,16 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 				return
 			}
 
+			for _, o := range orchestrators {
+				dbO, err := s.LivepeerNode.Database.SelectOrchs(&lpcommon.DBOrchFilter{
```
Comment (yondonfu) [inline server/webserver.go:942]: Might be worth adding a non-zero length check for the returned slice just to be safe. The length should usually be non-zero, but I think there could be an edge case where this endpoint is hit after a new address is added to the on-chain pool, but before the address is added to the DB by the orchestrator watcher.

**Review by yondonfu — CHANGES_REQUESTED** [review]

File: server/webserver.go (hunk 4 above)
Comment (kyriediculous) [reply server/webserver.go:942]: good point, if `len(dbO) == 0` we should set o.PricePerPixel to `big.NewRat(0, 1)` otherwise the floatstring conversion in the CLI will fail.

File: server/webserver.go (hunk 3 above)
Comment (kyriediculous) [reply server/webserver.go:951]: fixed in 6702d485

File: server/webserver.go (hunk 4 above)
Comment (kyriediculous) [reply server/webserver.go:942]: good point, if `len(dbO) == 0` we should set o.PricePerPixel to `big.NewRat(0, 1)` otherwise the floatstring conversion in the CLI will fail. 

Fixed in eac1d3b9

File: cmd/livepeer_cli/wizard_bond.go (hunk 1 above)
Comment (kyriediculous) [reply cmd/livepeer_cli/wizard_bond.go]: fixed in d74e79c68ae186fe7c2bcf3c3718296d0a7676b4

Comment (kyriediculous) [reply cmd/livepeer_cli/wizard_bond.go]: fixed in d74e79c68ae186fe7c2bcf3c3718296d0a7676b4

File: common/util.go (hunk 2 above)
Comment (kyriediculous) [reply common/util.go]: fixed in 926eac300b23364c9a1ed48df920837255d75838

moved function in subsequent fixup 80e013ac

Comment (kyriediculous) [reply common/util.go]: fixed in 926eac300b23364c9a1ed48df920837255d75838

moved function in subsequent fixup 80e013ac

File: server/webserver.go [hunk 5]
```diff
@@ -912,6 +912,21 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 				return
 			}
 
+			for _, o := range orchestrators {
```
Comment (yondonfu) [inline server/webserver.go:941]: Can we add a test for this update (`cliserver_test.go` might be a good place for the test to go)?

**Review by yondonfu — CHANGES_REQUESTED** [review]

File: server/cliserver_test.go [hunk 6]
```diff
@@ -86,3 +92,63 @@ func TestGetDelegatorInfo(t *testing.T) {
...
+	}
+
+	n, _ := core.NewLivepeerNode(eth, "./tmp", dbh)
+
+	s := NewLivepeerServer("127.0.0.1:1938", n)
+	mux := s.cliWebServerHandlers("addr")
+	srv := httptest.NewServer(mux)
+	defer srv.Close()
+
+	res, err := http.Get(fmt.Sprintf("%s/registeredOrchestrators", srv.URL))
```
Comment (yondonfu) [inline server/cliserver_test.go:148]: Let's add a test for when `SelectOrchs()` returns an empty slice resulting in the orchestrator's default `PricePerPixel` being set to `big.NewRat(0, 1)`?

**Review by yondonfu — CHANGES_REQUESTED** [review]

**Review by yondonfu — COMMENTED** [review]
Changes LGTM - let's squash the fixup commits.

**Comment by kyriediculous** [comment]
I rebased but CI still has issues, should be g2g though.

**Comment by kyriediculous** [comment]
CI is green , just need an approval !

**Review by yondonfu — APPROVED** [review]
LGTM! 🚢
